CRM/Financial/Form/FinancialType.php

```diff
@@ -120,10 +120,11 @@ public function postProcess() {
       foreach (['is_active', 'is_reserved', 'is_deductible'] as $field) {
         $params[$field] ??= FALSE;
       }
-      $financialType = civicrm_api4('FinancialType', 'save', [
-        'records' => [$params],
-        'checkPermissions' => TRUE,
-      ])->first();
+      $params['custom'] = CRM_Core_BAO_CustomField::postProcess($params,
```

Comment:
@colemanw is there an (easy) apiv4 way to take form-submitted values & pass to the apiv4?

Reply:
Not that I'm aware of. Such a function certainly would be handy.

Reply:
It would basically be the inverse of `\Civi\Api4\Generic\Traits\DAOActionTrait::formatCustomParams`

Reply:
Hmm intention was to entirely depend on APIv4 save action to update the FT details even its custom values, but that doesn't seem to be the case.